canvas_sdk/client/base.py

```diff
@@ -117,10 +117,15 @@ def call(action, url, request_context, params=None, data=None, max_retries=None,
             status_code = http_error.response.status_code
             # If we can't retry the request, raise a CanvasAPIError
             if status_code not in RETRY_ERROR_CODES or retry >= retries:
-                error_json = http_error.response.json()
+                try:
+                    error_json = http_error.response.json()
+                    message = str(error_json)
```

Comment:
This can just be `http_error.response.content` and save us the re-stringifying of the body, right?